nav/ui/common.py

```
@@ -0,0 +1,282 @@
+from typing import Callable, Optional, Any
```

Ruff (0.14.5)

Import from `collections.abc` instead: `Callable`

Import from `collections.abc`

(UP035)